src/settings-ui/Settings.UI/FlyoutWindow.xaml.cs

```diff
@@ -35,7 +35,7 @@ private void FlyoutWindow_Activated(object sender, Microsoft.UI.Xaml.WindowActiv
         {
             if (args.WindowActivationState == Microsoft.UI.Xaml.WindowActivationState.Deactivated)
             {
-                this.Hide();
```

Comment:
@donlaci FYI, we were hiding the window to make sure it was in memory and loaded only once. Showing/hiding it would be instant, and wouldn't require a reload. If it's instant with this change it's fine though 😊.

Reply:
@niels9001 Hi! Yes, it might happen, that I change it back to Hide().
Actually I am trying to reach that there should be allways one (if any) instance of the settings exe. Curreently it is possible to open many instances of it. I am working right on this topic.
But thanks for the remark!